labs.yml

```
@@ -133,3 +133,6 @@ commands:
     flags:
       - name: workspace_ids
         description: List of workspace IDs to create account groups from.
+
+  - name: migrate_locations
```

```suggestion
  - name: migrate-locations
```
Dashes, not underscores